### DISCUSSION TITLE
Animations with placeholders and animations.yml

### DISCUSSION THREAD

**Discussion by zFrank03** [Zrips/CMI#6189] [Feature Requests]
It would be very handy if CMI allowed you to create animations.

Animations should be created in the animations.yml file and used via the "%CMI_animations:<AnimationName>%" placeholder.
The file might be similar to neznamy's TAB file, which I find very handy:
![image](https://user-images.githubusercontent.com/64710654/134397095-43057cda-95e8-42bc-a94f-a2366558e7ba.png)

Unfortunately NEZNAMY TAB does not allow to use its animations in other plugins, I hope that with CMI this becomes possible.

**Comment by gecko10000** [comment]
The main issue with animations is that most places won't update often enough to keep up with that. In order to get a hologram with an animation you'd have to have it update more than once a second. However, you could probably accomplish what you want with a Javascript placeholder.

**Comment by Zrips** [comment]
Animated placeholders would require tracking of each player current placeholder state to give the correct result for next update. But at same time this might be completely out of wack if placeholder gets called more than once or from another place causing it to shift, for example using same animation in holograms and then in tablist would create inconsistent animations. While idea is good implementation would create to many inconsistencies and I would receive quite a few reports about this behavior even if I would make capitalized comment section on top of file informing about issue like this.
